lib/rubocop/cop/style/concat_array_literals.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Enforces the use of `Array#push(item)` instead of `Array#concat([item])`
+      # to avoid redundant array literals.
+      #
+      # @safety
+      #   This cop is unsafe, as it can produce false positives if the receiver
+      #   is not an `Array` object.
+      #
+      # @example
+      #
+      #   # bad
+      #   list.concat([item])
```

Comment:
I think it's better to add examples with multiple elements as well, so people won't think this works only one-element concats.

Reply:
Exactly! I've added examples.